showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITDynamicRoutingHeaders.java

```diff
@@ -175,6 +203,22 @@ public void testGrpc_matchesHeaderName() {
     assertThat(requestHeaders).containsExactlyElementsIn(expectedHeaders);
   }
 
+  @Test
+  public void testGrpc_implicitHeadersEnum() {
+
+    RepeatRequest request =
+        RepeatRequest.newBuilder().setInfo(ComplianceData.newBuilder().setFKingdomValue(5)).build();
+    RepeatResponse actualResponse = grpcComplianceClient.repeatDataSimplePath(request);
+    String headerValue = grpcComplianceInterceptor.metadata.get(REQUEST_PARAMS_HEADER_KEY);
+    assertThat(headerValue).isNotNull();
+    List<String> requestHeaders =
+        Arrays.stream(headerValue.split(SPLIT_TOKEN)).collect(Collectors.toList());
+    List<String> expectedHeaders =
+        ImmutableList.of(
+            "info.f_bool=false", "info.f_double=0.0", "info.f_int32=0", "info.f_kingdom=5");
```

Comment:
Can you add a comment above the expextedHeaders explaining that the fields besides `info.f_kingdom` are expected to be the default values (false, 0.0, 0) since the request's info message doesn't set the values?

Reply:
Done.